### DISCUSSION TITLE
Someone could help me on playing remote audio on iOS?

### DISCUSSION THREAD

**Discussion by devtelluria** [dooboolab/react-native-audio-recorder-player#351] [Q&A]
Hi!
I created a stackoverflow topic about my problem, and wonder if anyone could help me with it:
[https://stackoverflow.com/questions/68288629/play-remote-audio-with-react-native-audio-recorder-player-and-rnfetchblob-not-wo](https://stackoverflow.com/questions/68288629/play-remote-audio-with-react-native-audio-recorder-player-and-rnfetchblob-not-wo)
I appreciate any help!

**Comment by hyochan** [comment]
Looks like it has been answered!
